### PR TITLE
Add wide to long cli

### LINKED ISSUE #39: add entry point for wide to long conversion

### PR DESCRIPTION
**Pull request by stefpiatek**
Dependency on #38 as it uses the interface file from there. Overall didn't need to alter too much but found a couple of warnings from pandas, so altered the parsing to match that. Otherwise was done mostly using the internals. Have tested out on DSH and added documentation for how to set up data for export

Closes #39

### CODE REVIEW

File: src/rred_reports/redcap/README.md
Comment (harryjmoss) [inline src/rred_reports/redcap/README.md:1]: Thanks for adding this!

File: src/rred_reports/redcap/interface.py [hunk 1]
```diff
@@ -1,20 +1,63 @@
+from pathlib import Path
+
 import typer
+from loguru import logger
+
+from rred_reports.redcap.main import ExtractInput, RedcapReader
+from rred_reports.reports.interface import get_config
```
Comment (harryjmoss) [inline src/rred_reports/redcap/interface.py:7]: Should we define `get_config` in `src/rred_reports/__init__.py` maybe?

File: src/rred_reports/redcap/redcap_config.toml
Comment (harryjmoss) [inline src/rred_reports/redcap/redcap_config.toml:1]: 👍

**Review by harryjmoss — APPROVED** [review]

File: src/rred_reports/redcap/interface.py (hunk 1 above)
Comment (stefpiatek) [reply src/rred_reports/redcap/interface.py:7]: Yeah good call there, same with top_level_dir, I'll change that

File: tests/test_redcap_interface.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+
+@pytest.fixture()
+def set_top_level_dir() -> None:
+    """
+    Manually set the top level directory for running in CI, and revert it on teardown
+    """
+    original_value = interface.top_level_dir
+    interface.top_level_dir = Path(__file__).parents[1]
+    yield
```
Comment (harryjmoss) [inline tests/test_redcap_interface.py:17]: 👍

**Pull request merged**